apps/ledger-live-mobile/src/families/ethereum/EthereumCustomFees/Ethereum1559CustomFees.tsx

```diff
@@ -65,14 +73,36 @@ const Ethereum1559CustomFees = ({
         maxPriorityFeePerGas,
       },
     }));
+
   const { errors, warnings } = status;
   const { maxPriorityFee: maxPriorityFeeError, maxFee: maxFeeError } = errors;
-  const { maxPriorityFee: maxPriorityFeeWarning, maxFee: maxFeeWarning } =
-    warnings;
+  const { maxFee: maxFeeWarning } = warnings;
+  let { maxPriorityFee: maxPriorityFeeWarning } = warnings;
+
+  // give user a warning if maxFeePerGas is lower than pending transaction maxFeePerGas + 10% of pending transaction maxPriorityFeePerGas for edit eth transaction feature
+  if (
+    !maxPriorityFeeWarning &&
+    transactionRaw &&
+    transactionRaw.maxPriorityFeePerGas
```

Comment:
More concise and modern way to do this.
cf. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
```suggestion
    transactionRaw?.maxPriorityFeePerGas
```